### DISCUSSION TITLE
How to reupdate letsencrypt certificate

### DISCUSSION THREAD

**Discussion by carlosecv** [Tecnativa/doodba#406] [Q&A]
Someone knows how do i update lets encrypt certificate Im getting wrong certificate once I have duplicated one of my copier instances to new domain.

Regards

**Comment by joao-p-marques** [comment]
Sorry, I am not sure I understand the question: do you have 2 instances duplicated (with the same domains)? Or do they have different domains?

Are the domains correctly pointing to the appropriate server?

**Comment by carlosecv** [comment]
Its a new domain but I add the same old domain to the list:

domains_prod:
- hosts:
  - www.newdomain.com
  - newdomain.com
  - oldname.olddomain.com.mx
  - demo.newdomain.com
domains_test:
- hosts:
  - testoldname.olddomain.com.mx

I put down the old copier instance and up the new one and now i have the domain is insecure by incorrect certificate

**Comment by joao-p-marques** [comment]
Ok, but do you have those 2 instances on the same server? If not, to which IP is the domain pointing?

Anyway, this seems to be more of an issue/question regarding the usage of Traefik and let's encrypt, not with doodba-copier-template... 

I am moving this to https://github.com/Tecnativa/doodba/discussions

**Reply by carlosecv** [reply]
Yes this instances are on the same server

**Reply by joao-p-marques** [reply]
Ok. Well, that should probably be an error somewhere in the process related to Traefik and Let's Encrypt certificates. Can you inspect the Traefik logs to see what is happening? 
And also, can you make sure you haven't gone into any Let's Encrypt limits? E.g. is your new domain correctly pointing to the server?
Finally, did you try refreshing the browser to make sure it was picking up the correct new certificate?

**Reply by carlosecv** [reply]
Well I have being working on this, I will try to explain what happened:

I migrated My copier instance to a new server, so i run copier on new server with the same traefik file sample that is on docs folder of copier git project

Now on this new server Im getting that my CN is not anymore my domain name on ssl the certificate :

Common Name (CN)	TRAEFIK DEFAULT CERT

instead:

Common Name (CN)	www.mydomain.com

I dont think is a rate limit issue by lets encrypt but im stuck here because every actual domain with latest copier version now happens

I hope someone has some insight on this or maybe better, instead of use lets encript how to use a certificate files?

Im sorry if my questions are too basic or out of scope of copier project its because im not an expert on traefik.

Thank you

**Reply by joao-p-marques** [reply]
Ok, just to make sure, you mention that you migrated your doodba-copier-template instance to a new server, did you point the domains correctly to the new server?

@Yajo do you have any idea of what might be happening?

**Reply by yajo** [reply]
You probably have something wrongly configured.

If your Odoo was working properly in the old server, most chances is that you did something wrong configuring the server itself, Traefik or the DNS as @joao-p-marques says.

In any case, all of that goes outside the scope of doodba. You'll get better help in the Traefik forums probably.

**Reply by carlosecv** [reply]
Its correct Yajo.

This time it was a cache problem with web browsers or my equipment, yesterday it was incorrect the certificate today it worked all right.

Thank you very much for your advices.
